src/index.d.ts

```diff
@@ -210,7 +210,7 @@ export function createElement<
 	P extends JSXInternal.SVGAttributes<T>,
 	T extends HTMLElement
 >(
-	type: keyof JSXInternal.IntrinsicElements,
+	type: keyof JSXInternal.IntrinsicSVGElements,
```

Comment:
Sorry, drive-by change.

I couldn't find a situation in which this actually made a difference, as `createElement`/`h` seemingly always falls back to a very loose overload that allows anything, but we extracted out `IntrinsicSVGElements` a bit more than a year ago and so we should be using it.